### ISSUE TITLE
Limit size of images on emails

### ISSUE THREAD

**Issue by ItsiW** [Couchers-org/couchers#1079]
A very lovely photo of Jakub, but perhaps it doesn't need to be in such detail to extend over the margins of the email.

![image](https://user-images.githubusercontent.com/39325627/114479366-c2ceb280-9c43-11eb-90a5-814b7c3766f9.png)

**Comment by aapeliv** [comment]
Should be the thumbnail size, not full size. See also #871

**Comment by alexishonzik** [comment]
I'll work on this!

**Comment by alexishonzik** [comment]
We noticed in friend_request.md in backend/templates that the full_url is used. We were intending on changing this to the thumbnail_url, however we realized that this will cause the tests in backend/tests/test_email.py to fail because of the assertions for making sure the full_url is in the HTML (Line 150). Assuming we are understanding this correctly, can we change the tests to assert that the thumbnail URL is in the HTML instead of the full_url? Thanks!
